rudder-webapp/SOURCES/rudder-upgrade

```diff
@@ -359,14 +343,6 @@ upgrade_ncf() {
       sed -i "s%^loggers=\(.*\)%loggers=\1,log_rudder%" ${CONFIGURATION_REPOSITORY}/ncf/ncf.conf
     fi
 
-    # 2.9.0: Make sure that ncf uses the right CFEngine port (5309)
-    STEP="Make sure that ncf uses the right CFEngine port (5309)"
-
-    if ! cat ${CONFIGURATION_REPOSITORY}/ncf/ncf.conf | perl -p0e "s/\n/%%%%%/g" | egrep -q "classes.community_edition[^%]*%%%%%cfengine_port=5309"; then
-      cat ${CONFIGURATION_REPOSITORY}/ncf/ncf.conf | perl -p0e "s/\n/%%%%%/g" | sed "s/\(classes.community_edition[^%]*%%%%%cfengine_port=\)[0-9]\+/\15309/" | perl -p0e "s/%%%%%/\n/g" > ${CONFIGURATION_REPOSITORY}/ncf/ncf.conf.new
-      mv ${CONFIGURATION_REPOSITORY}/ncf/ncf.conf.new ${CONFIGURATION_REPOSITORY}/ncf/ncf.conf
-    fi
```

Comment:
oh, nice catch. it took me a while to understand that it was indeed not needed (ncf.conf has a lot of magic)